app/main/views.py

```diff
@@ -99,6 +165,43 @@ def get_service(service_id):
     return jsonify(services=jsonify_service(service))
 
 
+@main.route('/services-archive/<int:service_archive_id>', methods=['GET'])
+def get_archived_service(service_archive_id):
```

Comment:
OK, it's all renamed now - Table is ArchivedService, and all URLs are archived-services. Feels a lot nicer.